test_scripts/SDL5_0/MobileVersioning/PlayPauseButton/002_play_pause_v4_invalid_data.lua

```diff
@@ -42,7 +42,11 @@ config.application1.registerAppInterfaceParams.syncMsgVersion = {
 local function subscribeButtonSuccess(pButName)
   local cid = commonSmoke.getMobileSession():SendRPC("SubscribeButton", { buttonName = pButName })
   local appIDvalue = commonSmoke.getHMIAppId()
-  EXPECT_HMINOTIFICATION("Buttons.OnButtonSubscription", { appID = appIDvalue, name = "PLAY_PAUSE", isSubscribed = true })
+   commonSmoke.getHMIConnection():ExpectRequest("Buttons.SubscribeButton",
+    { appID = appIDvalue, buttonName = "PLAY_PAUSE" })
+  :Do(function(_, data)
+      commonSmoke.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+    end)
```

Comment:
In line `56` there is an expectation on `Buttons.OnButtonSubscription`. Suppose it needs to be replaced by `Buttons.UnsubscribeButton` request